backend/config/blockexplorer.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import "github.com/digitalbitbox/bitbox-wallet-app/backend/coins/coin"
+
+type BlockExplorer interface {
+	Enabled()
+	ExplorerURL()
+}
+
+type BlockExplorerConfig struct {
+	UseExplorer bool `json:"useCustomBlockExplorer"`
+	// Thies field is filled with a default when the config is created
+	// the default needs to be the placeholder value specified in the frontend.
+	ExplorerURL string `json:"explorerURL"`
+}
+
+// Implement custom blockexplorer for Bitcoin
```

Comment:
maybe `blockexplorer.go` should only define the interface and the implementation should live in the respective coin directory ? e.g. `coins/btc/blockexplorer.go` ?